evadb/udfs/ndarray/fuzzy_join.py

```diff
@@ -34,5 +34,7 @@ def forward(self, df: pd.DataFrame) -> pd.DataFrame:
             ret (pd.DataFrame): The cropped frame.
         """
         ret = pd.DataFrame()
-        ret["distance"] = df.apply(lambda row: fuzz.ratio(row[0], row[1]), axis=1)
+        ret["distance"] = df.apply(
+            lambda row: fuzz.ratio(str(row[0]), str(row[1])), axis=1
```

Comment:
So this lib only works for string?

Reply:
Yes. https://github.com/seatgeek/thefuzz